src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,125 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px;
+
+  &__row {
+    display: flex;
+    margin-bottom: 46px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:not(:last-child) {
+    margin-right: 48px;
+  }
+
+  &__image {
+    padding: 16px 3px 0;
+    height: 134px;
+    width: 160;
+
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $basic-text-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $price-and-code-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    line-height: 18px;
+    @extend %flex-align-center-space-between;
+  }
+
+  &__price-word {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $price-and-code-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $basic-text-color;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+    @extend %flex-align-center-space-between;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $basic-text-color;
+  }
+
+  &__button-buy {
+    height: 40px;
```

Comment:
please try to not hardcode height, you can try to use padding instead

Reply:
done